answers/Namespaces/2b901a20-9fc0-4191-b1bc-5fb2820cf71a/UserApp.php

```diff
@@ -0,0 +1,11 @@
+<?php
+namespace App;
+
+class User
+{
+    public function __construct()
+    {
+        echo "This is a user in the App namespace.";
+    }
```

Comment:
Adding an `echo` on the constructor feels wrong to me. This might teach learners bad habits on wrong usage of a constructor. Why not do
```
public static function showMessage()
{
  // Do your echo here.
}
```

Reply:
You can do this `App::Model::User::showMessage()` and `App::User::showMessage()` right?

Reply:
@TenshiLyle Yes, but this is not part of the content. Please check this PR https://github.com/CodingAvenue/codestop-course-php-oop/pull/15 for the whole content flow. Thank you.

Reply:
> You can do this `App::Model::User::showMessage()` and `App::User::showMessage()` right?

Errr this should be `\App\Model\User::showMessage()` and `\App\User::showMessage()`